client/my-sites/people/team-list/index.jsx

```diff
@@ -55,6 +57,21 @@ var Team = React.createClass( {
 			);
 		}
 
+		if ( this.props.fetchInitialized && ! this.props.users.length && this.props.role && ! this.props.fetchingUsers  ) {
```

Comment:
The `render` method is already rather long and adding another `if` doesn’t help much. How can we make this method more welcoming to newcomers?